Create-Cluster.md

```diff
@@ -3,66 +3,90 @@
 ## Prerequisites
 
 ```
-
+$ brew install kops
 $ brew install kubernetes-cli
 $ brew install kubernetes-helm
 $ brew install terraform
+$ git clone git@github.com:ministryofjustice/kubernetes-investigations.git
+$ git clone git@github.com:ministryofjustice/cloud-platform-environments.git
+$ git clone git@github.com:ministryofjustice/laa-fee-calculator.git
 
+Have access to the moj-cloud-platforms-dev Auth0 tenant 
 ```
 
+### Create a Kuberos application on Auth0
+
+1. Use your Github credentials to log into https://manage.auth0.com. Switch the tenant to `moj-cloud-platforms-dev`.
+
+1. Under `Applications`, create a new single ["Regular Web App"](https://auth0.com/docs/applications/webapps). Name the application `<your-cluster-name>-kuberos`. 
+
+1. Under `Settings`, take note of the app's "Client ID & Secret" and add `https://login.apps.<your-cluster-name>.k8s.integration.dsd.io/ui` to the `Allowed Callback URLs`.
+
+1. On Github, create an **org-owned** [Github Oauth app](https://auth0.com/docs/connections/social/github). Set the callback URL pointing to https://tenant-name.eu.auth0.com/login/callback and the Homepage URL to https://login.apps.your-cluster-name.k8s.integration.dsd.io. Take note of the Github "Client ID & Secret". This will be used in the next step.
```

Comment:
This came up yesterday when chatting with @ollieanwyll : it's only required if there's no tenant available (ie. if you're starting from scratch). We've currently been using a single tenant for all clusters so this won't be required as part of the recovery process.

However, it might be worth discussing whether we want a different tenant per cluster (I don't see any reason unless we were to use different auth0 rules per cluster, which is not the case currently).

It might be worth splitting into a different document? I think it's cause enough confusion :)